README.md

```diff
@@ -22,31 +20,90 @@ ODK Aggregate can be deployed on Google's App Engine, enabling users to quickly
 * ODK developer Slack archive: [http://opendatakit.slackarchive.io](http://opendatakit.slackarchive.io) 
 * ODK developer wiki: [https://github.com/opendatakit/opendatakit/wiki](https://github.com/opendatakit/opendatakit/wiki)
 
-## Release cycle
+## Getting the code
 
-TBD
+1. Fork the Aggregate project ([why and how to fork](https://help.github.com/articles/fork-a-repo/))
 
-## Setting up your development environment
+1. Clone your fork of the project locally. At the command line:
 
-1. Download and install [Git](https://git-scm.com/downloads) and add it to your PATH
+        git clone https://github.com/YOUR-GITHUB-USERNAME/aggregate
 
-1. Download and install [IntelliJ](https://www.jetbrains.com/idea/) 
+## Setting up the database
```

Comment:
Maybe we can add some instructions on how to set up PostgreSQL or provide a link to a tutorial which gives step by step instructions. Instructions on how to use `psql` command to create a database and warning for Windows users to set up the environment variables would be helpful for users who are new to PostgreSQL.

Reply:
I will try to find a good guide to add a link :)

Reply:
Back with the links:
- The [official download pages](https://www.postgresql.org/download/) for each system offers all the information one needs to download and install PostgreSQL
- The [PostgreSQL wiki pages](https://wiki.postgresql.org/wiki/Detailed_installation_guides) have some not quite up-to-date but useful guides